2025-04-06 05:43:18,764 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:43:18,767 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:43:18,769 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:43:21,189 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:43:21,192 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:43:21,195 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:43:23,352 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:43:23,356 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:43:23,360 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:43:25,576 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:43:25,579 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:43:25,582 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:43:28,284 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:43:28,287 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:43:28,290 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:43:31,694 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:43:31,698 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:43:31,701 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:43:36,854 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:43:36,858 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:43:36,861 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:43:45,307 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:43:45,310 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:43:45,313 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:44:00,082 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:44:00,085 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:44:00,088 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:44:27,690 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:44:27,693 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:44:27,697 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:45:21,056 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:45:21,059 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:45:21,063 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:51:42,437 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:51:42,440 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:51:42,443 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:52:44,317 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:52:44,321 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:52:44,324 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:53:46,459 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:53:46,463 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:53:46,466 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:54:48,732 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:54:48,736 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:54:48,741 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:55:50,633 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:55:50,636 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:55:50,640 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:56:52,878 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:56:52,881 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:56:52,884 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:58:17,824 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:58:17,827 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:58:17,831 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:58:21,144 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:58:21,147 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:58:21,151 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:58:23,820 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:58:23,823 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:58:23,826 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:58:27,078 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:58:27,081 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:58:27,084 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:58:30,862 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:58:30,865 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:58:30,868 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
2025-04-06 05:58:34,390 INFO: Contact Routing System startup [in /app/app.py:26]
2025-04-06 05:58:34,393 INFO: Loading model and data... [in /app/app.py:62]
2025-04-06 05:58:34,396 ERROR: Error loading model and data: [Errno 2] No such file or directory: '../models/routing_model.pkl' [in /app/app.py:83]
